CMakeLists.txt

```diff
@@ -289,7 +289,11 @@ if(GENERATE_DOC)
     file(COPY ${SOURCE_DIR}/docs/ldoc.css DESTINATION ${BUILD_DIR}/docs)
 
     add_custom_target(ldoc ALL
-        DEPENDS ${BUILD_DIR}/doc/index.html ${BUILD_DIR}/docs/ldoc.css)
+        DEPENDS
+            ${BUILD_DIR}/doc/index.html
+            ${BUILD_DIR}/docs/ldoc.css
+            generate_awesomerc
+    )
```

Comment:
I'm not sure if this change should be in this commit. It doesn't have anything to do with the index, I think. Perhaps the previous commit (which already has some unexplained dependency additions/fixes anyway).

Reply:
done